cirq/ops/moment_test.py

```diff
@@ -253,3 +253,11 @@ def test_inverse():
     assert cirq.inverse(m) == m**-1
     assert cirq.inverse(cirq.inverse(m)) == m
     assert cirq.inverse(cirq.Moment([cirq.measure(a)]), default=None) is None
+
+
+def test_immutable_moment():
+    with pytest.raises(AttributeError):
+        q1, q2 = cirq.LineQubit.range(2)
+        circuit = cirq.Circuit(cirq.X(q1))
+        moment = circuit.moments[0]
+        moment.operations += (cirq.Y(q2),)
```

Comment:
Add a newline.

Reply:
Where do you want me to add a line, at the end of the program?

Reply:
Yes.

Reply:
I see. Thanks.